cdc/sink/mysql/mysql_syncpoint_store.go

```diff
@@ -231,6 +232,22 @@ func (s *mysqlSyncPointStore) SinkSyncPoint(ctx context.Context,
 		return cerror.WrapError(cerror.ErrMySQLTxnError, err)
 	}
 
+	// set global tidb_external_ts to secondary ts
```

Comment:
Pls add some comments about the tidb's version which supports this variables

Reply:
I will add it later.